src/lake/Lake/Build/Facets.lean

```diff
@@ -95,6 +95,9 @@ builtin_facet oleanPrivateFacet @ olean.private : Module => FilePath
 /-- The `ilean` file produced by `lean`. -/
 builtin_facet ilean : Module => FilePath
 
+/-- The `ir` file produced by `lean` (with the module system enabled). -/
+builtin_facet ir : Module => FilePath
```

Comment:
@tydeu FYI, this is just a copy-paste for now but I assume will be relevant for landing the full `--setup`